e2e/testcafe-devextreme/tests/dataGrid/headerPanel.ts

```diff
@@ -151,3 +151,32 @@ test('Toolbar should render on changing visibility if visibility is false initia
     visible: false,
   },
 }));
+
+test('Toolbar should not reset its widget values when changing the disabled property', async (t) => {
+  const dataGrid = new DataGrid('#container');
+  const toolbar = dataGrid.getToolbar();
+  const textBox = toolbar.getItem(0);
+
+  // act
+  await t.typeText(textBox, 'test')
```

Comment:
@markallenramirez please add `;` at the end of the string, linter job in CI failed due to missed semicolon